### DISCUSSION TITLE
Missing flag in publish-package GitHub workflow?

### DISCUSSION THREAD

**Discussion by git-afsantos** [pyscaffold/pyscaffold#550] [Q&A] [answered]
Hello!

The last step of the `publish-package.yml` workflow runs the following commands to build and publish the package to PyPI.

```bash
python -m pip install --upgrade pip setuptools tox
python -m tox -e clean,build
python -m tox publish -- --verbose --repository pypi
```

The [last line](https://github.com/pyscaffold/pyscaffold/blob/de7aa5dc059fbd04307419c667cc4961bc9df4b8/.github/workflows/publish-package.yml#L33) in particular, seems to be missing the `-e` option.
Without it, `tox` would run the default environment, which, in this case, runs `pytest`.

I just wanted to discuss it here first, before submitting a bug report, because I am still new to tox, so I am not completely sure.

Thanks for putting the project together, by the way! :raised_hands:

**Comment by abravalheri** [comment, marked as answer]
Oops, I might have forgotten the flag last time I changed it.

Thank you very much André for pointing it out! You are correct, the '-e' should be there...

**Reply by abravalheri** [reply]
It seems however that tox managed to guess and do the right thing (I can see the latest releases were uploaded to PyPI). But this behaviour is undocumented and should not be relied upon...

I will not have access to a PC until next week, but if you would like to submit a PR with the fix, I would be more than happy to accept it!

Thanks again for the heads up!

**Reply by git-afsantos** [reply]
Great! I will prepare a PR, then. Glad I could help!
